app/src/main/kotlin/com/wire/android/ui/home/conversations/messages/ConversationMessagesViewModel.kt

```diff
@@ -92,6 +97,7 @@ class ConversationMessagesViewModel @Inject constructor(
         savedStateHandle.get<String>(EXTRA_CONVERSATION_ID)!!
     )
 
+    private var imageMessageOnFullscreen: UIMessage? = null
```

Comment:
I am not sure if this should leave inside ViewModel I think it is worth trying to move this inside a state related to this view by using remember{} and creating a state object for that UI element or for whole conversation screen

Reply:
ahh okay my bad ! :)